e2eAIOK/DeNas/nlp/bert_trainer.py

```diff
@@ -35,6 +39,15 @@ def _pre_process(self):
             custom_ops_thop = customer_ops_map_thop()
             macs_thop, _ = profile(self.model, inputs=(inputs,), custom_ops=custom_ops_thop)
             logging.info("(THOP) MACs: %.2f" % (macs_thop/(1000**3)))
+        if self.cfg.is_tl:
+            self.teacher_model = ModelBuilderNLPDeNas(self.cfg)._init_extra_model(self.cfg.teacher_model, self.cfg.teacher_model_structure)
+            self.teacher_distiller = kd.KD(pretrained_model=self.teacher_model, use_saved_logits=True)
+            self.logger.info("Successfully load teacher model!")
+            if self.cfg.is_saving_logits:
+                self.teacher_distiller.prepare_logits(self.train_dataloader, epochs=int(self.cfg.train_epochs))
+                self.logger.info("Successfully save teacher model logits!")
+                sys.exit()
```

Comment:
why do sys.exit() after saving logits? does that mean saving logits will definitely happened at the end of the process.

Reply:
Yes, it means the saving-logits has saved the needed logits in the function "prepare_logits", and no need for the next steps, so call the "sys.exit()".

Reply:
@tianyil1 , so can I understand as 'saving logits' and 'transferrable_model.make_transferrable_with_knowledge_distillation' is a if/else ? If user want to save logits, they may not to do transfer learning?

Reply:
adhoc meeting with @tianyil1 to clarify this issue, create an issue to fix later https://github.com/intel/e2eAIOK/issues/168